v2/emailpassword/serverless/with-aws-lambda/auth-serverless.mdx

```diff
@@ -5,16 +5,27 @@ hide_title: true
 ---
 
 <!-- COPY DOCS -->
-<!-- ./thirdpartyemailpassword/serverless/with-aws-lambda/auth-serverless.mdx -->
+<!-- ./emailpassword/serverless/with-aws-lambda/auth-serverless.mdx -->
```

Comment:
change it back to copying from thirdpartyemailpassword

Reply:
Done.